Eastern-Research-Group/mywaterway
# Feature/453 update dischargers layer

## Related Issues:
* [HMW-453](https://jira.epa.gov/browse/HMW-453)
* [HMW-460](https://jira.epa.gov/browse/HMW-460)

## Main Changes:
* Updated the ECHO service call to also pull back GPC (general permit) dischargers and a couple more fields (permit type and permit components).
* Updated the dischargers list view on the overview panel to be a virtualized list, because we now have a lot of data.
  * I didn't do this for the Identified Issues panel, because there aren't very many dischargers with significant effluent violations.
* Added the ability to filter dischargers by permit components.
* Fixed an issue with saving not working for the dischargers and monitoring location layers.
* Added an AGO filter disclaimer for saving the dischargers layer from the overview panel.
* Updated text for the save widget (from Kiki's email).

## Steps To Test:
1. Navigate to http://localhost:3000/community/dc/overview
2. Verify there are 306 dischargers (used to be 4)
3. Click on the permitted dischargers tab
4. Verify the filters work
5. Save to AGO 
6. Verify a filter disclaimer is shown in the description and the feature count in the web map matches HMW
7. Repeat steps 4 - 6 for the past water conditions on the monitoring tab
8. Navigate to http://localhost:3000/community/dc/identified-issues
9. Repeat steps 5 - 6
10. Verify text changes in the Save panel of the Add & Save Data Widget.
